satellytes/satellytes.com
# fix: adapt base url to netlify

### What was done?
The Base URL was previously adapted to gatsby cloud pr deployments. This PR changes that to the netlify pattern.

### How to verify?
Verify that on the [preview](https://deploy-preview-784--satellytes.netlify.app/) of this PR, the base URL of the meta images is the same as the preview URL:
<img width="405" alt="image" src="https://github.com/satellytes/satellytes.com/assets/78978542/885622a8-901c-43d7-94dc-394f6e7850f7">
